packages/mobile/src/account/LinkBankAccountScreen.tsx

```diff
@@ -53,7 +65,12 @@ function StepOne({ kycStatus }: Props) {
           <Text style={styles.action}>{t('linkBankAccountScreen.failed.title')}</Text>
           <Text style={styles.description}>{t('linkBankAccountScreen.failed.description')}</Text>
           <View style={styles.button}>
-            <PersonaButton kycStatus={kycStatus} text={t('linkBankAccountScreen.tryAgain')} />
+            <PersonaButton
+              kycStatus={kycStatus}
+              text={t('linkBankAccountScreen.tryAgain')}
+              onPress={onPressPersona}
+              onCancelled={() => setIsKycVerifying(false)}
+            />
           </View>
           <TouchableOpacity
```

Comment:
We should be using buttons that respect platform conventions: opacity on iOS and Ripple effect on Android.
`Button` / `TextButton` / `BorderlessButton` from our `react-components` folder should be preferred as they handle this directly.